content/enterprise_influxdb/v1.9/about-the-project/release-notes-changelog.md

```diff
@@ -15,35 +15,81 @@ The release of InfluxDB Enterprise 1.9 is different from previous InfluxDB Enter
 in that there is no corresponding InfluxDB OSS release.
 (InfluxDB 1.8.x will continue to receive maintenance updates.)
 
+#21015: build: upgrade to go 1.15.10
+
 ### Features
 - Support user-defined *node labels*.
   Node labels let you assign arbitrary key-value pairs to meta and data nodes in a cluster.
   For instance, an operator might want to label nodes with the availability zone in which they're located.
 - Improve performance of `SHOW SERIES CARDINALITY` and `SHOW SERIES CARDINALITY from <measurement>` InfluxQL queries.
   These queries now return a `cardinality estimation` column header where before they returned `count`.
-- Improve diagnostics for license problems. Add [license expiration date](/enterprise_influxdb/v1.9/features/clustering-features/#entitlements) to `debug/vars` metrics.
+- Improve diagnostics for license problems.
+  Add [license expiration date](/enterprise_influxdb/v1.9/features/clustering-features/#entitlements) to `debug/vars` metrics.
 - Add improved [ingress metrics](/enterprise_influxdb/v1.9/administration/config-data-nodes/#ingress-metric-by-measurement-enabled--false) to track points written by measurement and by login.
```

Comment:
allows for collection of statistics of points, values AND new series written per measurement and by login.

Previously we had points and values.  But the new series has been added...along with the measurement and login options.  Further...this data is collected at the data node level.  These metrics are exposed for each data node and operators can use these to aggregate the write requests across the entire cluster to gain a detailed understanding of the growth of series within a measurement and what user credentials are being used to write data to the cluster and more!